services/todo-legacy/repositories/todo.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  AnyObject,
+  DataSourceConstructor,
+  DefaultCrudRepository,
+  Filter,
+  Model,
+  ModelBaseConstructor,
+  ObjectType,
+  Repository,
+  Where,
+  juggler
+} from '@loopback/repository';
+import { inject, Application } from '@loopback/core';
+import * as _ from 'lodash';
+export class TodoRepository {
+  // FIXME(kev): The juggler.ModelBase type doesn't carry any definitions
+  // for the CRUD functions that typically get mixed in when createModel
+  // makes a definition, so we have to use type "any" to avoid complaint!
+  // model: juggler.ModelBase | any;
+  model: juggler.PersistedModel | any;
```

Comment:
As of the current version I've got installed of `@loopback/repository`; the `DataSourceConstructor` returns an object of type `juggler.ModelBase`. Removing the any will throw compilation errors since the object is not strictly of that type.

If there's a newer version of that library that fixes this problem, then we should release it to npmjs (I'm not linking the packages so that I can avoid CI failures)